### ISSUE TITLE
300VW dataset download link

### ISSUE THREAD

**Issue by ghost** [grigorisg9gr/robust_deformable_face_tracking#1]
I have tried many times to download 300-VW dataset in http://ibug.doc.ic.ac.uk/resources, Unfortunately failed,could you have a link to download this dataset?
Thanks

**Comment by patricksnape** [comment]
That link is the correct link. However, it probably fails because you end up losing the connection to the server (because the file is so large [1.7GB] - this becomes more likely). If you want to download the file and resume failures of connection that I suggest you use `wget` which will continue partial downloads. Unfortunately, the file is protected on the server and requires you to give your details in order to proceed. However, getting around this is actually quite easy:

```
wget --header "Cookie: csrftoken=TOKEN" \
--post-data "csrfmiddlewaretoken=TOKEN&first_name=YOURNAME&last_name=YOURNAME&email_address=YOUREMAIL%40email.com&affiliation=YOURAFFILIATION" \
--save-cookies cookies.txt \
http://ibug.doc.ic.ac.uk/download/300VW_Dataset_2015_12_14.zip/
```

Fill in your first name, last name, email (%40 is the escaped @ symbol) and affiliation. The TOKEN variable is the hard part. Open the browser to the web page (using private browsing probably) and go to the page where it asks you to fill in your information. In Google Chrome, right click on the submit button and click "Inspect". Then,  look for the element above the submit button element called `<div style="display:none"></div>` which contains an element `<input type="hidden" name="csrfmiddlewaretoken" value="TOKEN">` where TOKEN is the value we need to replace on the POST data and Cookie data above.

After this, you will have a new file called `cookies.txt` in the directory where you ran the command. You can then run:

```
wget -c --load-cookies cookies.txt \
http://ibug.doc.ic.ac.uk/download/300VW_Dataset_2015_12_14.zip/
```

Which should download the file - the special part being that we load the cookies we previously received from the page and the fact that we use the `-c` flag which will continue the download it fails!

**Comment by ghost** [comment]
I did as your advise but failed to download, the zip file is only 1.1G,but I found this file is 1.7G

**Comment by patricksnape** [comment]
I'm not sure why you are having so much trouble, this file has been downloaded successfully over 1000 times. Perhaps try downloading it somewhere else with a more stable connection? Or maybe your ISP is killing large file downloads?

**Comment by grigorisg9gr** [comment]
If you are still facing any trouble, you can download it from [here](http://bit.ly/1r41sTX).

**Comment by ghost** [comment]
I am in China, and I have asked many people in China and they meet the same problem

**Comment by patricksnape** [comment]
Did you try the new link?

**Comment by zarchary-10** [comment]
I meet the same problem,and the link in dropbox is failed

**Comment by patricksnape** [comment]
The files are now broken into multiple smaller downloads - are you still having trouble?

**Comment by grigorisg9gr** [comment]
​If you cannot download it from the site, please find here a new link: http://bit.ly/2blx7Le

**Comment by zarchary-10** [comment]
The new link is ok，Thank you for your reply @grigorisg9gr

**Comment by chaimahajtaher** [comment]
Can you share with me the new link?

**Comment by AbhinavUtkarsh** [comment]
Doesn´t work for me either from Europe, even with the wget, gets stuck right at 1GB multiple times. Don´t think my internet is bad.

```
(base) abhi@abhi-blade:~/Desktop/datasets/keyboxmatching$ wget -c --load-cookies cookies.txt \
http://ibug.doc.ic.ac.uk/download/300VW_Dataset_2015_12_14.zip/
URL transformed to HTTPS due to an HSTS policy
--2024-04-20 15:35:41--  https://ibug.doc.ic.ac.uk/download/300VW_Dataset_2015_12_14.zip/
Resolving ibug.doc.ic.ac.uk (ibug.doc.ic.ac.uk)... 146.169.50.44
Connecting to ibug.doc.ic.ac.uk (ibug.doc.ic.ac.uk)|146.169.50.44|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 1853231339 (1,7G) [application/zip]
Saving to: ‘index.html’

index.html           58%[==========>         ]   1,00G  --.-KB/s    in 22m 23s 

2024-04-20 15:58:05 (785 KB/s) - Connection closed at byte 1079422619. Retrying.

--2024-04-20 15:58:06--  (try: 2)  https://ibug.doc.ic.ac.uk/download/300VW_Dataset_2015_12_14.zip/
Connecting to ibug.doc.ic.ac.uk (ibug.doc.ic.ac.uk)|146.169.50.44|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 1853231339 (1,7G) [application/zip]
Saving to: ‘index.html’

index.html           27%[====>               ] 489,03M  --.-KB/s    in 27m 40s 

2024-04-20 17:32:23 (302 KB/s) - Read error at byte 1079422619/1853231339 (Connection timed out). Retrying.

--2024-04-20 17:32:25--  (try: 3)  https://ibug.doc.ic.ac.uk/download/300VW_Dataset_2015_12_14.zip/
Connecting to ibug.doc.ic.ac.uk (ibug.doc.ic.ac.uk)|146.169.50.44|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 1853231339 (1,7G) [application/zip]
Saving to: ‘index.html’

index.html           58%[==========>         ]   1,00G  --.-KB/s    in 27m 36s 

2024-04-20 18:00:03 (637 KB/s) - Connection closed at byte 1079549595. Retrying.

--2024-04-20 18:00:06--  (try: 4)  https://ibug.doc.ic.ac.uk/download/300VW_Dataset_2015_12_14.zip/
Connecting to ibug.doc.ic.ac.uk (ibug.doc.ic.ac.uk)|146.169.50.44|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 1853231339 (1,7G) [application/zip]
Saving to: ‘index.html’

index.html           58%[==========>         ]   1,00G  --.-KB/s    in 27m 55s 

2024-04-20 18:28:02 (629 KB/s) - Connection closed at byte 1079549595. Retrying.

--2024-04-20 18:28:06--  (try: 5)  https://ibug.doc.ic.ac.uk/download/300VW_Dataset_2015_12_14.zip/
Connecting to ibug.doc.ic.ac.uk (ibug.doc.ic.ac.uk)|146.169.50.44|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 1853231339 (1,7G) [application/zip]
Saving to: ‘index.html’

index.html           58%[==========>         ]   1,00G  --.-KB/s    in 28m 4s  

2024-04-20 18:56:11 (626 KB/s) - Connection closed at byte 1079549595. Retrying.

--2024-04-20 18:56:16--  (try: 6)  https://ibug.doc.ic.ac.uk/download/300VW_Dataset_2015_12_14.zip/
Connecting to ibug.doc.ic.ac.uk (ibug.doc.ic.ac.uk)|146.169.50.44|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 1853231339 (1,7G) [application/zip]
Saving to: ‘index.html’

index.html           58%[==========>         ]   1,00G  --.-KB/s    in 20m 3s  

2024-04-20 19:16:19 (875 KB/s) - Connection closed at byte 1079549595. Retrying.

--2024-04-20 19:16:25--  (try: 7)  https://ibug.doc.ic.ac.uk/download/300VW_Dataset_2015_12_14.zip/
Connecting to ibug.doc.ic.ac.uk (ibug.doc.ic.ac.uk)|146.169.50.44|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 1853231339 (1,7G) [application/zip]
Saving to: ‘index.html’

index.html           58%[==========>         ]   1,00G  --.-KB/s    in 24m 11s 

2024-04-20 19:40:36 (726 KB/s) - Connection closed at byte 1079549595. Retrying.
``
